tests/security/audit/ausearch.pm

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Verify the "ausearch" utility can search the audit log file for certain events using various keys or
+#          other characteristics of the logged format
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $tmp_output = '/tmp/out';
+    my $tmp_backup = '/tmp/backup';
+
+    select_console 'root-console';
+
+    # Make sure audit service is started
```

Comment:
@ShawnHa0 
As I remembered, auditd is up by default, so just checking it is in active status is fine?

or you can start it if it is now up there

Reply:
Good point! I changed auditd checking from 'start' to 'is-active'.

Reply:
cool